_includes/v2.1/app/common-steps-secure.md

```diff
@@ -0,0 +1,84 @@
+## Step 2. Start a cluster
+
+Before we can run code, we need to start a cluster.  Follow the steps below to start a secure local cluster with 3 nodes.
+
+1. Create a CA (certificate authority) that will be used to sign all of your other certificates.
+
+    {% include copy-clipboard.html %}
+    ~~~ sh
+    $ cockroach cert create-ca --certs-dir=/tmp/certs--certs-dir=/tmp/certs --ca-key=/tmp/certs/ca.key
+    ~~~
+
+2. Create a certificate and key for each node in the cluster.  Since we are running a local cluster, we will cheat a bit by re-using the same certificate and key for each node.
```

Comment:
Well, we aren't really cheating. We are just being technically correct :) I think we can simply say: "Since we are running a local cluster, we can reuse..."